scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -914,6 +959,53 @@ s32 get_nr_online_cpus(void)
 	return cpus;
 }
 
+static int init_allowed_cpus(void)
+{
+	struct bpf_cpumask *mask;
+	int err = 0;
+
+	bpf_rcu_read_lock();
```

Comment:
Hmm... can you please explain the locking a bit? I don't understand what rcu read locks are protecting.

Reply:
Thanks @htejun, the rcu read lock is definitely not needed, the goal was just to make the verifier happy.

I've pushed a new change removing the rcu locking both from the `allowed_cpumask` initialization and, more important, from `pick_idle_cpu()`. We still need it to call `bpf_cpumask_set_cpu()`, but that is just affecting the initialization, so basically there's not extra locking involved at runtime, which is nice.